zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -920,4 +920,26 @@ mod test {
         begin_tx_deploy(0x1020304050607080u64);
         begin_tx_deploy(0xfffffffffffffffeu64);
     }
+
+    #[test]
+    fn create_tx_for_existing_account() {
+        let address = Address::repeat_byte(23);
+        let nonce = 10;
+        let new_address = get_contract_address(address, nonce + 1);
+
+        let ctx = TestContext::<1, 2>::new(
+            None,
+            |accs| {
+                accs[0].address(address).nonce(nonce).balance(eth(10));
+            },
+            |mut txs, _| {
+                txs[0].from(address).to(new_address).value(eth(2));
+                txs[1].from(address);
```

Comment:
Some comments would be helpful. 
```suggestion
                txs[0].from(address).to(new_address).value(eth(2)); // Initialize new_address with some balance and an empty code hash
                txs[1].from(address); // Run a CREATE tx on new_address
```